src/Atmos/Model/source.jl

```diff
@@ -68,5 +73,8 @@ function atmos_source!(s::RayleighSponge, m::AtmosModel, source::Vars, state::Va
     coeff_top = s.c_sponge * (sinpi(FT(1/2)*(z - s.zsponge)/(s.zmax-s.zsponge)))^FT(4)
     coeff = min(coeff_top, 1.0)
   end
-  source.ρu -= state.ρu * coeff
-end
+
+  u = state.ρu / state.ρ
+  u_relax_state = SVector(s.u_ref, s.v_ref, s.w_ref)
+  source.ρu -= state.ρ * coeff * (u - u_relax_state)    
```

Comment:
```suggestion
  source.ρu -= state.ρ * coeff * (u - u_relaxed)    
```
w.r.t. comment on line `78`